### DISCUSSION TITLE
Would it be possible to have some sort of color correction shader?

### DISCUSSION THREAD

**Discussion by Contra-S** [profi200/open_agb_firm#108] [Ideas]
I was wondering if it would be possible to have an option to color correct the screen's colors, as some GBA emulators do with shaders like "gba-color".

**Comment by profi200** [comment]
I wanted to implement this for a while but there is one huge problem. These shaders all use the fragment shader feature of GPUs but the GPU on 3DS does not have this. I won't rule out that i find a way to make it happen but for now i have no way to accurately implement this.
